packages/js/plugins/http/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "@web3api/http-plugin-js",
+    "version": "0.0.1-prealpha.1",
+    "license": "MIT",
+    "main": "build/index.js",
+    "scripts": {
+      "build": "rimraf ./build && tsc",
+      "lint": "eslint --color --ext .ts src/",
+      "test": "jest --passWithNoTests --runInBand --verbose",
+      "test:watch": "jest --watch --passWithNoTests --verbose"
+    },
+    "dependencies": {
+      "@web3api/client-js": "0.0.1-prealpha.1",
+      "@web3api/core-js": "^0.0.1-prealpha.1",
+      "axios": "0.21.1"
+    },
+    "devDependencies": {
+      "@types/jest": "26.0.8",
+      "jest": "26.2.2",
+      "rimraf": "3.0.2",
+      "ts-jest": "26.1.4",
+      "ts-node": "8.10.2",
+      "typescript": "3.9.7"
```

Comment:
@dOrgJelli Maybe we can move all those common dev dependencies to root package.json?

Reply:
I'd rather keep them here so that we can move these packages out of the monorepo in the future w/o worrying about finding all deps.